src/main/scala/io/github/sentenza/hacktoberfest18/Main.scala

```diff
@@ -1,5 +1,49 @@
 package io.github.sentenza.hacktoberfest18
+import sort.mergeSort
```

Comment:
We don't need to import the mergeSort into our Main class, do we?

Reply:
yes we don't, it was bad merge, I tried to return it to its original state (the way you changed it yesterday)